recipes/civetweb/all/patches/0001-fix-install-bundle.patch

```diff
@@ -0,0 +1,13 @@
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
+@@ -257,9 +257,7 @@ if (CIVETWEB_ENABLE_SERVER_EXECUTABLE)
+         install(
+             TARGETS civetweb-c-executable
+             EXPORT ${PROJECT_NAME}-targets
+-            ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+-            LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+-            RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
++            DESTINATION ${CMAKE_INSTALL_BINDIR}
```

Comment:
so, `ARCHIVE`, `LIBRARY` and `RUNTIME` aren't needed?
I would expect something like:
```
            ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
            LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
            RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
            BUNDLE DESTINATION ${CMAKE_INSTALL_BINDIR}

```

Reply:
Useless for an executable